build-plugins/build-support/src/main/kotlin/aws/sdk/kotlin/gradle/plugins/artifactsizemetrics/PutArtifactSizeMetricsInCloudWatch.kt

```diff
@@ -34,60 +34,49 @@ internal abstract class PutArtifactSizeMetricsInCloudWatch : DefaultTask() {
     @TaskAction
     fun put() {
         val currentTime = Instant.now()
-        val pluginConfig = this.project.rootProject.extensions.getByType(ArtifactSizeMetricsPluginConfig::class.java)
-        val release = project.property("release").toString().let {
-            check(it.isNotEmpty()) { "The release property is empty. Please specify a value." } // "-Prelease=" (no value set)
-            it
+        val pluginConfig = project.rootProject.extensions.getByType(ArtifactSizeMetricsPluginConfig::class.java)
+        val releaseTag = project.property("release").toString().also {
```

Comment:
correctness: `.property(...)` will throw an exception if the property is not set. I think you want to use `findProperty` and then handle the potential null / empty value.